webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -181,7 +180,15 @@
                             </tr>
                             <tr>
                                 <th scope="row">Build Architecture Info</th>
-                                <td>{{ report.cpu_info }}</td>
+                                <td>
+                                    {% if report.cpu_info %}
+                                        {% if report.cpu_info.startswith('AuthenticAMD family 20 model 1') or
+                                              report.cpu_info.startswith('AuthenticAMD family 20 model 2') %}
```

Comment:
family 20 won't have any models added and even if it did they should have the bug fixed.